src/etc/testcases/taskdefs/optional/junitlauncherwithtag.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0"?>
```

Comment:
Can you include these test targets in the `junitlauncher.xml` itself instead of a new one just for testing tags?